### PR TITLE
Feat (#485) : added feature to close the active tab with middle mouse click

### PR DESCRIPTION
**Pull request by Sahilm416**
-> feat #485 
-> Added the functionality to close the active tab with middle mouse click
-> Middle click functionality is added inside the whole main section

### CODE REVIEW

File: packages/bruno-app/src/components/RequestTabPanel/index.js [hunk 1]
```diff
@@ -150,7 +155,7 @@ const RequestTabPanel = () => {
       <div className="pt-4 pb-3 px-4">
         <QueryUrl item={item} collection={collection} handleRun={handleRun} />
       </div>
-      <section className="main flex flex-grow pb-4 relative">
+      <section className="main flex flex-grow pb-4 relative" onMouseDown={(e) => MiddleClickClose(e)}>
```
Comment (helloanoop) [inline packages/bruno-app/src/components/RequestTabPanel/index.js]: Please rename `MiddleClickClose` as `handleMouseDown`

You can add a comment in the callback like this.
You also don't have to write an explicit return.
 
```javascript
const handleMouseDown = (e) => {
  // middle click
  if (e.button === 1) {
    handleCloseClick(e, focusedTab, dispatch);
  }
};
```

File: packages/bruno-app/src/components/RequestTabPanel/index.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import RunnerResults from 'components/RunnerResults';
 import VariablesEditor from 'components/VariablesEditor';
 import CollectionSettings from 'components/CollectionSettings';
 import { DocExplorer } from '@usebruno/graphql-docs';
-
+import { handleCloseClick } from 'components/RequestTabs/RequestTab/index';
```
Comment (helloanoop) [inline packages/bruno-app/src/components/RequestTabPanel/index.js]: Let'd not wire components like this.
You need to use a redux action.

**Review by helloanoop — CHANGES_REQUESTED** [review]

**Comment by helloanoop** [comment]
Also @Sahilm416 We have a functionality where - if a user is editing something, upon clicking in close we show an alert modal requesting to either save or cancel the draft.

The middle click implementation should ensure that this flow is not broken.

**Comment by Sahilm416** [comment]
Thanks for the review !

**Pull request closed without merging**